.github/DISCUSSION_TEMPLATE/request-help.yml

```diff
@@ -10,32 +19,14 @@ body:
   - type: input
     id: self-hosted-version
     attributes:
-      label: If you're self-hosting Renovate, tell us what version of Renovate you run.
-    validations:
-      required: false
-
-  - type: dropdown
-    id: self-hosted-platform
-    attributes:
-      label: If you're self-hosting Renovate, select which platform you are using.
-      options:
-        - 'AWS CodeCommit'
-        - 'Azure DevOps (dev.azure.com)'
-        - 'Azure DevOps Server'
-        - 'Bitbucket Cloud (bitbucket.org)'
-        - 'Bitbucket Server'
-        - 'Gitea or Forgejo'
-        - 'github.com'
-        - 'GitHub Enterprise Server'
-        - 'gitlab.com'
-        - 'GitLab self-hosted'
+      label: If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.
```

Comment:
Keep this simple/vague. We should have an auto action describing exactly the details we need in case they are incomplete

Reply:
Having a action/bot comment later makes following the repository  noisier.

In the old situation:

- chance for user to fill out information correctly right away
- ask for clarification often

In the new situation:

- no chance for user to fill out information correctly
- ask for clarification via label

I think you already thought about this, and decided you like the bot label approach?